src/app/dashboard/whitelist/components/DatePickerSingleInput.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+// TODO @davekaj: get this file merged into the polymath-ui repo, and
+// remove the local version of it from here
```

Comment:
if one line is not enough to describe TODO use
`
// TODO @davekaj: get this file...
// TODO ...merged into
`
Otherwise IDE recognizes second string as a simple comment, not todo

Reply:
makes sense. fixed